SU-HSDO/suhumsci
# Arch 150 last menu dropdown off page

# READY FOR REVIEW

# Summary
- ARCH-150
- ARCH-155
- ARCH-171

# Steps to Test
1. Visit the dev site
1. Analyze the main dropdown menu and confirm that ARCH-150 is resolved
1. Analyze the side menu and confirm that ARCH-171 is resolved
1. Edit the home page content, confirming that there is a new "underline-accent" class, and that applying it to a bit of text has the desired effect (per ARCH-155)

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [ARCH-150](https://stanfordits.atlassian.net/browse/ARCH-150)
- [ARCH-155](https://stanfordits.atlassian.net/browse/ARCH-155)
- [ARCH-171](https://stanfordits.atlassian.net/browse/ARCH-171)
